include/upipe/uref_pic.h

```diff
@@ -171,6 +172,27 @@ static inline int uref_pic_plane_clear(struct uref *uref,
                                 hsize, vsize, fullrange);
 }
 
+/** @see ubuf_split_fields */
+static inline int uref_split_fields(struct uref *uref, struct uref **odd,
+        struct uref **even)
+{
+    int ret = UBASE_ERR_ALLOC;
+
+    *odd = uref_dup_inner(uref);
+    *even = uref_dup_inner(uref);
+
+    if (*odd && *even)
+        ret = ubuf_split_fields(uref->ubuf, &(*odd)->ubuf, &(*even)->ubuf);
+
+end:
```

Comment:
Is this label a leftover of something or did you leave something out when committing?  I think you've missed freeing odd or even should the allocation of the other fail.